### DISCUSSION TITLE
Gitea PHP API as a Super Power in JCB

### DISCUSSION THREAD

**Discussion by Llewellynvdm** [joomengine/Joomla-Component-Builder#952] [Announcements]
Just finished the complete Gitea PHP API  back-end that is also [fully container](https://git.vdm.dev/joomla/gitea#vdm-joomla-gitea-service) ready. Here its packaged https://git.vdm.dev/joomla/gitea as a "super-power" which is one of the formats JCB exports and imports (will soon roll this out to you all). These classes are automatically exported from JCB with their own UML for each class. We will soon add normal composer packaging export options to [JCB](https://git.vdm.dev/joomla/Component-Builder). Thanks to GPT-4  yes it only took two days to build this [Gitea API](https://git.vdm.dev/joomla/gitea) with GPT-4 and JCB

The new super-powers area is going to empower all of you to use PHP in a complete liberating way in Joomla, the boundaries has been removed....

We will be rolling out this new feature of JCB in a few days, it is yet another step towards Joomla 4, as it forms a critical part of the new architecture needed.

**Comment by DinaClinebellQ97** [comment]
Wow! Looking forward to it!
